wled00/FX_fcn.cpp

```diff
@@ -1349,7 +1361,8 @@ void WS2812FX::service() {
         // overwritten by later effect. To enable seamless blending for every effect, additional LED buffer
         // would need to be allocated for each effect and then blended together for each pixel.
         [[maybe_unused]] uint8_t tmpMode = seg.currentMode();  // this will return old mode while in transition
-        delay = (*_mode[seg.mode])();         // run new/current mode
+        frameDelay = (*_mode[seg.mode])();         // run new/current mode
+        if (frameDelay < speedLimit) frameDelay = FRAMETIME;                    // limit effects that want to go faster than target FPS
```

Comment:
This should involve `_frametime` not _speedLimit_.
`_frametime` reflects desired FPS (rounded).

Reply:
speed limit is actually 85% of _frametime. I've added it to prevent that effects that return a custom FRAMETIME exceed user defined target FPS. 85% to have some margin for "acdeptable" hoverspeed that can be handled.